aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,14 +1,34 @@
 ---
 remote_endpoint: ~
-name: "v1.8"
+name: "v1.9"
 proposals:
-  - name: upgrade_framework
+  - name: step_1_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework to v1.8"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.8"
+      title: "Multi-step proposal to upgrade mainnet framework to v1.9"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.9"
     execution_mode: MultiStep
     update_sequence:
       - DefaultGas
```

Comment:
@msmouse , just to double check, does this look ok to you?

Reply:
yes.